File: CS/13_Hyperlink/ModifyHyperlink.cs

using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using Spire.Presentation;

namespace ModifyHyperlink
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();            
        }

        private void btnRun_Click(object sender, EventArgs e)
        {
            //Create a PowerPoint document.
            Presentation presentation = new Presentation();

            //Load the file from disk.
            presentation.LoadFromFile(@"..\..\..\..\..\..\Data\Template_Ppt_5.pptx");

            //Find the hyperlinks you want to edit.
            IAutoShape shape = (IAutoShape)presentation.Slides[0].Shapes[0];

            //Edit the link text and the target URL.
            shape.TextFrame.TextRange.ClickAction.Address = "http://www.e-iceblue.com";
            shape.TextFrame.TextRange.Text = "E-iceblue";

            String result = "Result-ModifyHyperlink.pptx";

            //Save to file.
            presentation.SaveToFile(result, FileFormat.Pptx2013);

            //Launch the PowerPoint file.
            PptDocumentViewer(result);
        }

        private void PptDocumentViewer(string fileName)
        {
            try
            {
                System.Diagnostics.Process.Start(fileName);
            }
            catch { }
        }
    }
}

File: VB/13_Hyperlink/ModifyHyperlink.vb
Imports System.ComponentModel
Imports System.Text
Imports Spire.Presentation

Namespace ModifyHyperlink
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub btnRun_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnRun.Click
			'Create a PowerPoint document.
			Dim presentation As New Presentation()

			'Load the file from disk.
			presentation.LoadFromFile("..\..\..\..\..\..\Data\Template_Ppt_5.pptx")

			'Find the hyperlinks you want to edit.
			Dim shape As IAutoShape = CType(presentation.Slides(0).Shapes(0), IAutoShape)

			'Edit the link text and the target URL.
			shape.TextFrame.TextRange.ClickAction.Address = "http://www.e-iceblue.com"
			shape.TextFrame.TextRange.Text = "E-iceblue"

			Dim result As String = "Result-ModifyHyperlink.pptx"

			'Save to file.
			presentation.SaveToFile(result, FileFormat.Pptx2013)

			'Launch the PowerPoint file.
			PptDocumentViewer(result)
		End Sub

		Private Sub PptDocumentViewer(ByVal fileName As String)
			Try
				Process.Start(fileName)
			Catch
			End Try
		End Sub
	End Class
End Namespace